justspacesapp/views.py

```diff
@@ -30,3 +35,23 @@ class SurveyList(ListView):
     model = FormEntry
     template_name = "survey_list.html"
     context_object_name = 'surveys'
+
+class Signup(FormView):
+    template_name = "registration/signup.html"
+    form_class = JustSpacesUserCreationForm
+    success_url = '/'
+
+    def post(self, request):
+        superuser = JustSpacesUser(is_superuser=True)
+        form = self.form_class(request.POST, instance=superuser)
+        print("Here is the form:")
+        print(form)
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get('username')
+            raw_password = form.cleaned_data.get('password1')
+            user = authenticate(username=username, password=raw_password)
+            login(request, user)
+            return redirect('home')
```

Comment:
I'd prefer that users do not get automatically logged into the site. Let's redirect to login.

Note! I also think we want users who sign-up to verify their accounts via email. I modified the relevant issue accordingly: https://github.com/datamade/just-spaces/issues/28

Do you agree?

Reply:
Sure! I'll comment in the issue.